src/frontend/src/components/Posts.svelte

```diff
@@ -114,4 +117,8 @@
  .post {
     background-color:rgb(246, 245, 245);
  }
+
+ .npub {
+    color:darkgray;
+ }
```

Comment:
túto css klasu zmazať, [v Tailwinde je na to trieda](https://tailwindcss.com/docs/text-color) (odhadujem `text-gray-500`), to isté tá `post` trieda, v Tailwinde sa to robí cez `bg-gray-300`.